### ISSUE TITLE
Is this work in a webworker ?

### ISSUE THREAD

**Issue by JonathanDaSilva** [systemjs/plugin-babel#26]
Hi.
I have an angular2 application that I try to convert to webworker. 

But when Systemjs try to load an es6 file I have this error.
```
Error loading http://localhost:8000/systemjs-babel-build as "systemjs-babel-build" from http://localhost:8000/jspm_packages/npm/systemjs-plugin-babel@0.0.8/plugin-babel.js
```

Everything works fine when I load the same application without the webworker.

I have try to understand what's going on, you seems to load two different files for the browser and node.

Maybe in worker mode the wrong one is loaded ?

If I missed something feel free to ask me.

PS: Sorry for my bad english :(

**Comment by guybedford** [comment]
Yes, this is something that may need looking into to properly support. Will get to it when I can.

**Comment by JonathanDaSilva** [comment]
Thank a lot for your work.

**Comment by guybedford** [comment]
I'm not sure this is a bug actually. The fact that it is loading `http://localhost:8000/systemjs-babel-build` and not anything else would be due to the configuration. Are you definitely including the SystemJS configuration after loading SystemJS, that sets a map config for `systemjs-babel-build`?

**Comment by JonathanDaSilva** [comment]
Okay so I have checked. I was absolutely including the two config files (the browser and normal one), but they weren't the systemjs-babel-build in it. I have not noticed it because the page was loading without it on no-worker mod.

So I have add the map to the file into the config.

```
    "plugin-babel": "npm:systemjs-plugin-babel@0.0.8",
    'systemjs-babel-build': 'npm:systemjs-plugin-babel@0.0.8/systemjs-babel-browser.js',
```

I have one concerne about adding things manualy like that into this file, this will be updated in the future in case of an update or I have to update it everytime manualy.

Thanks a lot for your quick response, and sorry to bother you due to this stupid mistake I haven't checked my config because that was working without the worker…

**Comment by guybedford** [comment]
This is a bug, you're completely right.

I've added a fix in https://github.com/systemjs/plugin-babel/commit/8ff3bc56f856300338ca398f34eb5e3b1cae92c8.

**Comment by JonathanDaSilva** [comment]
I was not saying that was the fault of the plugin but maybe mine :D.

But anyway thank lot, this will be easier for the next people.

PS: I leave this open, feel free to close it.

**Comment by guybedford** [comment]
I will keep this open until the fix is released.

**Comment by guybedford** [comment]
I've also created https://github.com/systemjs/systemjs/issues/1186 to expand on the underlying cause.

**Comment by guybedford** [comment]
This has now been released in 0.0.9.
